velox/exec/fuzzer/AggregationFuzzerBase.cpp

```diff
@@ -565,11 +565,17 @@ void printStats(const AggregationFuzzerBase::FunctionsStats& stats) {
 std::string makeFunctionCall(
     const std::string& name,
     const std::vector<std::string>& argNames,
-    bool sortedInputs) {
+    bool sortedInputs,
+    bool distinctInputs) {
   std::ostringstream call;
-  call << name << "(" << folly::join(", ", argNames);
+  call << name << "(";
   if (sortedInputs) {
-    call << " ORDER BY " << folly::join(", ", argNames);
+    call << folly::join(", ", argNames) << " ORDER BY "
```

Comment:
folly::join(", ", argNames) repeats many times. Consider extracting a temp variable.

Reply:
Fixed, along with the disabling of spill in `TableScan` in verifyDistinctAggregation.